Makefile

```diff
@@ -133,6 +133,7 @@ cleancerts:
 
 certs/CAkey.pem:
 	@echo No CA keypair present, generating
+	@mkdir -p certs
```

Comment:
makefile pro tip
```
certs/CAkey.pem: | certs
   ...

certs:
     @mkdir -p certs
```
this makes the _presence_ of `certs/` a prereq for certs/CAKey.pem, but make won't be picky about timestamps